load.php

```diff
@@ -291,6 +291,17 @@ function perflab_maybe_set_object_cache_dropin() {
 		return;
 	}
 
+	/**
+	 * Filters whether the Perflab server timing drop-in should be set.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param bool Whether the server timing drop-in should be set.
+	 */
+	if ( apply_filters( 'perflab_object_cache_dropin', '__return_false' ) ) {
```

Comment:
Maybe name this to `perflab_disable_object_cache_dropin`? Also the default value is incorrect, should be just the boolean.
```suggestion
	if ( apply_filters( 'perflab_disable_object_cache_dropin', false ) ) {
```